Automattic/jetpack
# packages: Add dev keywords for Composer

## Proposed changes:

Modern versions of Composer will automatically suggest passing `--dev` to `composer require` if certain keywords ("dev", "testing", or "static analysis") are set on the package. Let's set those keywords.

And then we may as well set additional keywords on them too since we're doing it.

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Proofread, see if the keywords chosen make sense.
* Not sure the `--dev` recommending can be tested without this being merged and released. 🤷
